packages/terra-divider/src/Divider.scss

```diff
@@ -0,0 +1,9 @@
+:local {
+  .divider {
+    border-color: var(--terra-horizontal-divider-border-color, #b2b5b6);
+    border-style: solid;
+    border-top-width: var(--terra-horizontal-divider-border-width, 1px);
```

Comment:
There's a few left over variables / files containing `horizontal` that should be updated after the name change. A quick search through the pull request for the word `horizontal` highlights a few